instrumentation/jmx-metrics/javaagent/src/main/resources/jmx/rules/camel.yaml

```diff
@@ -0,0 +1,262 @@
+        type: gauge
+        unit: ms
+        desc: Indicates the mean processing time, in milliseconds, for all exchanges processed since context start-up or the last reset operation.
+      MinProcessingTime:
+        metric: processing.min_time
+        unit: ms
+        type: gauge
+        desc: Indicates the shortest time, in milliseconds, to process an exchange since context start-up or the last reset operation.
+      LastProcessingTime:
+        metric: processing.last_time
+        unit: ms
+        type: gauge
+        desc: Indicates the time, in milliseconds, it took to process the last exchange.
+      DeltaProcessingTime:
+        metric: processing.delta_time
+        type: gauge
+        unit: ms
+        desc: Indicates the difference, in milliseconds, of the Processing Time of the last two exchanges transited the context.
+      TotalProcessingTime:
+        metric: processing.time
+        type: counter
+        unit: ms
+        desc: Indicates the total processing time, in milliseconds, to process all exchanges since context start-up or the last reset operation.
+      Redeliveries:
+        metric: redelivery.count
+        type: counter
+        unit: "{redeliveries}"
+        desc: Number of redeliveries happened since context start-up or the last reset operation.
+
+# Route Level metrics
+
+  - bean: org.apache.camel:context=*,type=routes,name=*
+    prefix: camel.route.
+    metricAttribute:
+      route: beanattr(RouteId)
+      context: param(context)
+    mapping:
+      ExchangesCompleted:
+        metric: exchange.completed.count
+        type: counter
+        unit: "{exchanges}"
+        desc: Indicates the total number of exchanges the route has processed successfully since route start-up or the last reset operation.
+      ExchangesFailed:
+        metric: exchange.failed.count
+        type: counter
+        unit: "{exchanges}"
+        desc: Indicates the total number of exchanges that the route has failed to process since route start-up or the last reset operation.
+      ExchangesInflight:
+        metric: exchange.inflight.count
+        type: updowncounter
+        unit: "{exchanges}"
+        desc: Indicates the number of exchanges currently transiting the route.
+      ExchangesTotal:
+        metric: exchange.count
+        unit: "{exchanges}"
+        type: counter
+        desc: Indicates the total number of exchanges, passed or failed, that the route has processed since route start-up or the last reset operation.
+      FailuresHandled:
+        metric: failure.handled.count
+        unit: "{exchanges}"
+        type: counter
+        desc: Indicates the number of exchanges failed and handled by an ExceptionHandler in the route.
+      ExternalRedeliveries:
+        metric: external.redelivery.count
+        unit: "{redeliveries}"
+        type: counter
+        desc: The total number of all external redeliveries since the route start-up or the last reset operation.
```

Comment:
sorry, I know naming isn't fun, after seeing these I'm not sure suffixing `.count` on everything is great, or at least it doesn't match existing semconv counter names, check out https://github.com/open-telemetry/semantic-conventions/issues/260